sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldData.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Azure.AI.FormRecognizer.Models
+{
+    /// <summary>
+    /// The details of field elements that are part of a <see cref="FormField"/>.
+    /// </summary>
```

Comment:
Any particular reason for updating the description? It seemed more descriptive before.
```
/// A form content element representing text that is part of a <see cref="FormField"/>.
/// This includes the location of the text in the form and a collection of the content
/// elements that make up the text.
```

Reply:
Not really. More to delete in case it changed in the future. 
Adding back